docs/dotnet/minimal-api.md

```diff
@@ -44,6 +44,7 @@
 - [felipeoriani/MinimalBlogApi](https://github.com/felipeoriani/MinimalBlogApi)
 - [bradygaster/TodoApp](https://github.com/bradygaster/TodoApp)
 - [davidfowl/TodoApi](https://github.com/davidfowl/TodoApi)
+- [ricardodemauro/URLShortnerMinimalApi](https://github.com/ricardodemauro/URLShortnerMinimalApi)
```

Comment:
Thanks for the contribution. But there are similar projects showing a minimal API approach here with the most popularity.